Easydict/objc/ViewController/Window/WindowManager/EZWindowManager.m

```diff
@@ -921,6 +921,9 @@ - (void)closeFloatingWindowIfNotPinnedOrMain:(EZWindowType)windowType {
 }
 
 - (void)closeFloatingWindowIfNotPinned:(EZWindowType)windowType exceptWindowType:(EZWindowType)exceptWindowType {
+    if (_fixedWindow == nil && _miniWindow == nil) {
```

Comment:
This solution is not good. This condition is very strange to be placed here. It is just to handle a special situation. 

In addition, it will cause the main window to not be closed.

Reply:
I fixed it in 481191cfafadef65656c4df3a81c6aa27c22e38f